packages/blog/articles/new-environments.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Creators: Verify Your Environment Metadata'
+summary: We've overhauled the environment metadata on Modrinth, and all creators must verify their settings.
+date: 2025-08-28T00:00:00-07:00
+authors: ['Dc7EYhxG']
```

Comment:
nitpick: you put your name at the bottom, just put yourself here alongside whoever did the backend

Reply:
won't include @AlexTMjugador unless he wants to be credited on the post. don't want to make it look like he's saying these words if he hasn't even read them